### PR TITLE
Included Nuxt 3 Demo project 

### PR DESCRIPTION
**Pull request by jaybharadia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@
 If you're using Directus in an open source project, you're very welcome to link this project here.
 
 - [Official Examples](https://github.com/directus/examples) - Integration examples with Directus.
+- [Nuxt 3 Demo](https://github.com/bryantgillespie/nuxt3-directus-starter)
```
Comment (bryantgillespie) [inline README.md:93]:
```suggestion
- [Nuxt 3 Demo](https://github.com/bryantgillespie/nuxt3-directus-starter) Opinionated Nuxt 3 / Directus Starter with Tailwind CSS
```

Comment (bryantgillespie) [reply README.md:93]: Let's just make it known that is a fairly opinionated starter for Directus / Nuxt 3.

Comment (jaybharadia) [reply README.md]: Great suggestion 👌 . Updated the PR @bryantgillespie .

File: README.md [hunk 2]
```diff
@@ -90,6 +90,7 @@
 If you're using Directus in an open source project, you're very welcome to link this project here.
 
 - [Official Examples](https://github.com/directus/examples) - Integration examples with Directus.
+- [Nuxt 3 Demo](https://github.com/bryantgillespie/nuxt3-directus-starter) – Opinionated Nuxt 3 / Directus Starter with Tailwind CSS
```
Comment (paescuj) [inline README.md:93]:
```suggestion
- [Nuxt 3 Demo](https://github.com/bryantgillespie/nuxt3-directus-starter) - Opinionated Nuxt 3 / Directus Starter with Tailwind CSS
```

File: README.md [hunk 3]
```diff
@@ -90,6 +90,7 @@
 If you're using Directus in an open source project, you're very welcome to link this project here.
 
 - [Official Examples](https://github.com/directus/examples) - Integration examples with Directus.
+- [Nuxt 3 Demo](https://github.com/bryantgillespie/nuxt3-directus-starter) - Opinionated Nuxt 3 / Directus Starter with Tailwind CSS
```
Comment (paescuj) [inline README.md:93]:
```suggestion
- [Nuxt 3 Demo](https://github.com/bryantgillespie/nuxt3-directus-starter) - Opinionated Nuxt 3 / Directus Starter with Tailwind CSS.
```

**Pull request merged**